packages/prosess-beregningsgrunnlag/src/components/beregningsresultatPanel/BeregningsResultatPanel.jsx

```diff
@@ -171,22 +193,34 @@ const createPeriodeResultat = (vilkaarBG, periodeData, lagPeriodeHeaders, intl,
     <React.Fragment key={`Wr${key}`}>
       {periodeData && lagPeriodeHeaders && lagPeriodeOverskrift(periodeData.headers, periodeIndex)}
       {!ikkeOppfylt && lagTabellRader(periodeData, ikkeVurdert)}
-      {ikkeOppfylt && lagTabellRaderIkkeOppfylt(periodeData.rowsAndeler, intl, halvGVerdi, key)}
+      {ikkeOppfylt && lagTabellRaderIkkeOppfylt(periodeData.rowsAndeler, intl, halvGVerdi, key, fagsakYtelseType)}
     </React.Fragment>
   );
 };
 const BeregningsresutatPanel = ({ intl, vilkaarBG, periodeResultatTabeller, halvGVerdi }) => {
   const skalLagePeriodeHeaders = periodeResultatTabeller.length > 1;
   return (
-    <Panel className={beregningStyles.panelRight}>
-      <Element className={beregningStyles.avsnittOverskrift}>
-        <FormattedMessage id="Beregningsgrunnlag.BeregningTable.Tittel" />
-      </Element>
-      <VerticalSpacer eightPx />
-      {periodeResultatTabeller.map((periodeData, index) =>
-        createPeriodeResultat(vilkaarBG, periodeData, skalLagePeriodeHeaders, intl, halvGVerdi, index),
+    <BeregningContext.Consumer>
```

Comment:
Kan også bruke useContext hook, men skjønner hvis det gir problemer med enzyme 😬

Reply:
takk for tips 😄